application/src/main/java/dev/stratospheric/todoapp/config/CustomConfigurationProperties.java

```diff
@@ -24,7 +24,7 @@ class CustomConfigurationProperties {
   private Boolean autoConfirmCollaborations;
 
   @NotEmpty
-  private String breadcrumbTableName;
+  private String environment;
```

Comment:
there's no need to inject the table name anymore, as we have this hardcoded at the entity class and only required this property to create the table.